### PR TITLE
Adds the Still to the Bar.

### PR DESCRIPTION
**Pull request by Iamgoofball**
This PR adds the Still to the bar. The still can be used with any food item to produce several kinds of alcoholic beverages that the crew would love to get drunk on.

Weed Vodka
Deep Fried Monkey's Delight Ale
Clown's Tears Whiskey

These are a few examples of what the still can be used for.

This also fixes the food duplication bug with the oven and such.

### CODE REVIEW

File: code/game/machinery/kitchen/oven.dm [hunk 1]
```diff
@@ -3,15 +3,15 @@
 	desc = "Cookies are ready, dear."
 	icon = 'icons/obj/cooking_machines.dmi'
 	icon_state = "oven_off"
+	var/orig = "oven"
+	var/production_meth = "cooking"
```
Comment (Atskadan) [inline code/game/machinery/kitchen/oven.dm:7]: omg it makes meth wtf!!!!!!!!!!!!!!

**Comment by ghost** [comment]
> var/production_meth

I had to double take. My common sense kicked in eventually but I almost thought we were getting meth labs for some reason.

**Comment by Atskadan** [comment]
also it would be cooler if you could make normal alcohols out of this instead of just "lel penis sandwich vodka!"

**Comment by Iamgoofball** [comment]
i can limit it to grown items if you want.

File: code/game/machinery/kitchen/oven.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/game/machinery/kitchen/oven.dm:7]: add meth labs 2014

**Comment by ghost** [comment]
I'd like if you did that Goof.

**Comment by Iamgoofball** [comment]
is done.

**Pull request merged**